perceval/utils/statevector.py

```diff
@@ -54,10 +55,12 @@ def _fockstate_sub(self, other):
     return StateVector(self) - other
 
 def _fockstate_pow(self, power: int):
-    bs = self.__copy__()
-    for i in range(power - 1):
-        bs = bs * self
-    return bs
+    if power < 0:
```

Comment:
I think you don't need this if, but anyway, this function is to disappear soon because it's been reimplemented C++-side (right?).

Reply:
We need this in case the user does something like bs**-1
But yes, this bit is gonna be replace by the c++ method